### ISSUE TITLE
Periodic boundary in unit cell problem in #49.

### ISSUE THREAD

**Issue by Hejie-Guo** [rquey/neper#172]
Hi,
I am trying to generate a periodic unit cell with the command you answer in #49.
But when I use the command "neper -M 50grains.tess -elttype tet -order 1 -meshqualmin 1.0 -rcl 1 -pl 1000 -o 50grains_tet -format geof" I get the false message as following:

```
======================== N e p e r =======================
Info : A software package for polycrystal generation and meshing.
Info : Version 4.1.2
Info : Built with: gsl|muparser|opengjk|openmp|nlopt
Info : Running on 2 threads.
Info : https://neper.info
Info : Copyright (C) 2003-2020, and GNU GPL'd, by Romain Quey.
Info : No initialization file found (/home/guohejie/.neperrc').
Info : ---------------------------------------------------------------
Info : MODULE -M loaded with arguments:
Info : [ini file] (none)
Info : [com line] 50grains.tess -elttype tet -order 1 -meshqualmin
       1.0 -rcl 1 -pl 1000 -o 50grains_tet -format geof
Info : --------------------------------------------------------------
Info : Reading input data...
Info : - Reading arguments... Info : Loading input data...
Info : - Loading tessellation...
Info : [i] Parsing file 50grains.tess'...
Info : [i] Parsed file `50grains.tess'.
Info : Meshing...
Info : - Preparing... (cl = 0.1357) 100%
Info : - 0D meshing... 100%
Info : - 1D meshing... 100%
Info : - 2D meshing... 100% (0|0.49/94%| 5%| 1%)
Info : - Fixing 2D-mesh pinches...
Info : - 3D meshing... Error : > Meshing of poly 1 (1) failed
Aborted (core dumped)
```

**Comment by Hejie-Guo** [comment]
Hello, 

I installed the old neper version 3.5.2 and the problem is solved.

**Comment by Hejie-Guo** [comment]
Hello, 

I have solve the problem with old version neper. But the results of the so called periodic RVE is no what expected to be. Is there any way to generate a cubic RVE with periodic boundary ? Cubic RVE is a general RVE to be used in abaqus.  
![1](https://user-images.githubusercontent.com/61571563/113717353-1dd14880-971e-11eb-92aa-4ac912f7bc5d.png)

**Comment by rquey** [comment]
There was a bug in version 4.1.2, which has been fixed in the latest version (currently 4.1.3-9).

As for getting a cubic RVE, see #100.
